tools/slo-rules-generator/slo-domains.yaml.example

```diff
@@ -1,4 +1,5 @@
-userportal-reports: # domain-name
+userportal-reports-v6: # config-name
+  domain: userportal-reports
```

Comment:
The `domain` could default to the config name -- to stay backwards compatible...

Reply:
Fixed in 1fcde6ac95f9cb9c1101b86092fb64e53ff70cce